script/upgrades/MU03/MU03.sol

```diff
@@ -142,21 +144,33 @@ contract MU03 is IMentoUpgrade, GovernanceScript {
   }
 
   function proposal_addEUROCToPartialReserve() private {
+    // addCollateralAsset will throw if it's already added
+    if (IReserve(partialReserveProxy).isCollateralAsset(bridgedEUROC) == false) {
+      transactions.push(
+        ICeloGovernance.Transaction(
+          0,
+          partialReserveProxy,
+          abi.encodeWithSelector(IReserve(0).addCollateralAsset.selector, bridgedEUROC)
+        )
+      );
+    }
+  }
+
+  function proposal_updateBiPoolManagerImplementation() public {
```

Comment:
I don't think we should set BiPoolManager implementation and BreakerBox in the same function. 
```suggestion
  function proposal_updateBiPoolManagerImplementation() public {
```

Reply:
updated